src/views/language/LanguageTable.tsx

```diff
@@ -112,6 +159,112 @@ const LanguageTable: React.FC = () => {
           sortParam: SortBy.PopulationOfDescendents,
           columnGroup: 'Population',
         },
+        {
+          key: 'Vitality (Meta)',
+          label: 'Vitality (Meta)',
+          render: (lang) => {
+            const { score, explanation } = computeVitalityMetascore(lang);
+            const b = scoreBucket(score);
+            return (
+              <Hoverable hoverContent={explanation}>
+                <div
+                  style={{
+                    display: 'flex',
+                    alignItems: 'center',
+                    gap: '0.35em',
+                    color: bucketColor(b),
+                  }}
+                >
+                  {bucketIcon(b)}
+                  <span style={{ fontVariantNumeric: 'tabular-nums' }}>{scoreLabel(score)}</span>
+                </div>
+              </Hoverable>
+            );
+          },
```

Comment:
I realize that this file is getting too big. Can you move the markdown to its own class `LanguageVitalityCell`? It will also help with code reuse.

Thanks for including the hoverable :) I like it.

```suggestion
          render: (lang) => {
            <LanguageVitalityCell lang={lang} type={VitalityMeterType.Metascore} />
            );
          },
```